### DISCUSSION TITLE
Novità

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#189] [Idee e proposte]
A me sembra che manchi proprio l'atto di apporre la propria firma prima di inserire l'argomento e i compiti. Non mi sembra quasi di  firmare quando vado ad inserire la lezione,secondo me dovrebbe essere come in altri registri, ovvero prima si firma la propria ora e solo una volta firmata la propria ora ,è possibile inserire la lezione e i compiti,oltre che le presenze e i voti .Inoltre se è possibile sarebbe bello quando il docente va ad inserire i compiti che ci siano quattro celle dove scrivere:1 una in cui l'argomento della lezione è visibile dai genitori/alunni affianco un'altra dove l'argomento è visibile solo al docente stesso e così per i compiti,un po' come visto nel registro argo 

per quanto riguarda le 4 celle allego foto per far capire meglio 

https://user-images.githubusercontent.com/93823125/155841884-9daa6bc5-5e1f-4806-9c34-cb003ea594ec.png

**Comment by scuolagioco** [comment]
Anche l'aggiunta della possibilità di segnare il giorno di consegna dei compiti non sarebbe male

**Comment by manuelbarberi** [comment]
si, infatti

**Comment by manuelbarberi** [comment]
risulterebbe tutto più comodo

**Reply by scuolagioco** [reply]
Anche quella di firmare ore di lezione in compresenza avendo attribuite materie diverse, ad esempio nella mia scuola la maggior parte dei casi a partire dal triennio , noi docenti ci troviamo ad essere in compresenza con l'ITP che ha un'altra materia attribuita e non quella teorica come capita a me.

**Reply by manuelbarberi** [reply]
Si può mi sembra firmare in compresenza.... basta cliccare su modifica

**Reply by trinko** [reply]
> Anche quella di firmare ore di lezione in compresenza avendo attribuite materie diverse, ad esempio nella mia scuola la maggior parte dei casi a partire dal triennio , noi docenti ci troviamo ad essere in compresenza con l'ITP che ha un'altra materia attribuita e non quella teorica come capita a me.

Se hai una compresenza con l'ITP la materia è la stessa: basta configurarle così nelle cattedre. In questo modo si può firmare la compresenza.
